src/Microsoft.IdentityModel.Clients.ActiveDirectory/UserAssertion.cs

```diff
@@ -44,7 +44,7 @@ public UserAssertion(string assertion) : this(assertion, OAuthGrantType.JwtBeare
         }
 
         /// <summary>
-        /// Constructor to create credential with client id, assertion and assertionType
+        /// Constructor to create credential with assertion and assertionType
```

Comment:
We probably want to link to a doc where this flow is explained? As a user I've always found these overload confusing. We need to help the developers who are not Identity geeks, there :)

Reply:
we do not have flows explanations in source code, so we might add link to https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-on-behalf-of , but since it is web resource It might become unavailable later

Reply:
provided sample